sources/packages/web/src/composables/useFormioDropdownLoader.ts

```diff
@@ -67,6 +67,30 @@ export function useFormioDropdownLoader() {
     );
   };
 
+  // Get Program description for the selected program and set to the field (programDesc) in formio
+  const loadProgramDesc = async (
+    form: any,
+    programId: number,
+    fieldId: string,
+  ) => {
+    const valueToBeLoaded = await EducationProgramService.shared.getStudentEducationProgram(
+      programId,
+    );
+    formioUtils.setComponentValue(
+      form,
+      fieldId,
+      `<h3><span style="background-color:transparent;color:#2965c5;"><strong>${valueToBeLoaded.name}</strong></span></h3>${valueToBeLoaded.description}
+      <table style="width:100%">
```

Comment:
why `<table>` ?

Reply:
Without table, it looks weird and we had to use &nbsp for the spaces, this is to be in sync with the wireframe. If you have suggestions, please tell.

Reply:
<img width="973" alt="image" src="https://user-images.githubusercontent.com/62901416/135680832-6a532ed7-3257-4bd7-b445-243592f76f4e.png">

Reply:
https://getbootstrap.com/docs/4.0/layout/grid/, you can use bootstrap classess

Reply:
So what's the difference?

Reply:
IMO, it's not appropriate to use the table for these types of styles, in future if we want to add a custom styles or override an actual table, it will be difficult and break styles

Reply:
OKay, let me change it.

Reply:
I do not see a reason to keep this HTML on Vue application. Can you please move this HTML to the form definition?